### PR TITLE
Markdig migration

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

**Comment by mairaw** [comment]
I double checked IntelliSense and most of the changes are harmless (trimming spaces from the end) or improvements. There might still be a few bare URLs that should be updated but I think we should finish the migration now and deal with any broken links as they show up in the CATS reports.

File: xml/System/DateTime.xml [hunk 1]
```diff
@@ -5226,12 +5226,12 @@ The following example parses an array of date strings by using the conventions o
...
-|`s` contains the Z or GMT time zone designator, and `styles` includes the <xref:System.Globalization.DateTimeStyles.RoundtripKind?displayProperty=nameWithType>.|None.|<xref:System.DateTimeKind.Utc>| 
+| If | Time zone conversion | Kind property |
+|----|----------------------|---------------|
+| `s` contains no time zone information. | None. | <xref:System.DateTimeKind.Unspecified?displayProperty=nameWithType> |
+| `s` contains time zone information. | To the time in the local time zone | <xref:System.DateTimeKind.Local?displayProperty=nameWithType> |
+| `s` contains time zone information, and `styles` includes the <xref:System.Globalization.DateTimeStyles.AdjustToUniversal?displayProperty=nameWithType> flag. | To Coordinated Universal Time (UTC) | <xref:System.DateTimeKind.Utc?displayProperty=nameWithType> |
+| `s` contains the Z or GMT time zone designator, and `styles` includes the <xref:System.Globalization.DateTimeStyles.RoundtripKind?displayProperty=nameWithType>. | None. | <xref:System.DateTimeKind.Utc> |
```
Comment (rpetrusha) [inline xml/System/DateTime.xml]:
```suggestion
| `s` contains the Z or GMT time zone designator, and `styles` includes the <xref:System.Globalization.DateTimeStyles.RoundtripKind?displayProperty=nameWithType>. | None. | <xref:System.DateTimeKind.Utc?displayProperty=nameWithType> |
```

File: xml/System.Threading.Tasks/TaskFactory.xml [hunk 2]
```diff
@@ -621,7 +621,11 @@ The following example creates a cancellation token, which it passes to separate
         <returns>The new continuation task.</returns>
         <remarks>
           <format type="text/markdown"><![CDATA[
-The NotOn* and OnlyOn* <xref:System.Threading.Tasks.TaskContinuationOptions>, which constrain for which <xref:System.Threading.Tasks.TaskStatus> states a continuation will be executed, are illegal with `ContinueWhenAll`.
+
```
[the remark this answers is not in the document]
Comment (mairaw) [reply xml/System.Threading.Tasks/TaskFactory.xml]: This is only true for the overloads that have the continuationOptions param.

File: xml/System.Xml/XmlNode.xml [hunk 3]
```diff
@@ -280,16 +280,16 @@ Display the modified XML...
...
 |NodeType|Name|BaseURI|  
 |--------------|----------|-------------|  
-|Document|#document|http://localhost/mydata.xml|  
-|DocumentType|Mydata|http://localhost/mydata.xml<br /><br /> The <xref:System.Xml.XmlDocumentType.SystemId%2A?displayProperty=nameWithType> or <xref:System.Xml.XmlDocumentType.PublicId%2A?displayProperty=nameWithType> properties can be used to identify where the DTD file was loaded from.|  
-|Element|baa|http://localhost/mydata.xml|  
-|Entity|xyz|http://localhost/doctype.dtd|  
-|EntityReference|xyz|http://localhost/mydata.xml|  
-|Attribute|woof|http://localhost/mydata.xml|  
+|Document|#document|`http://localhost/mydata.xml`|  
+|DocumentType|Mydata|`http://localhost/mydata.xml`<br /><br /> The <xref:System.Xml.XmlDocumentType.SystemId%2A?displayProperty=nameWithType> or <xref:System.Xml.XmlDocumentType.PublicId%2A?displayProperty=nameWithType> properties can be used to identify where the DTD file was loaded from.|
```
[the remark this answers is not in the document]
Comment (mairaw) [reply xml/System.Xml/XmlNode.xml]: I'm seeing it:
image

**Pull request merged**